src/VisualStudio/CSharp/Impl/Options/AdvancedOptionPageStrings.cs

```diff
@@ -214,6 +214,9 @@ public static string Option_Suggest_usings_for_types_in_reference_assemblies
         public static string Option_Suggest_usings_for_types_in_NuGet_packages
             => CSharpVSResources.Suggest_usings_for_types_in_NuGet_packages;
 
+        public static string Option_Add_usings_on_paste
+            => CSharpVSResources.Add_missing_using_directives_on_paste;
```

Comment:
option name and loc name should match (except the former can be prefixed with `Option_`.